### PR TITLE
Allow a student to see their submission even if they are not on a team

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Show a student their past submission, even if they are no longer on a team for a team ORA.

**What changed?**

- Changed behaviors for when a student views a team ORA when they are no longer on a team for the assignment:
   - Student has previously submitted: show their submission.
   - Student has not previously submitted: show the "this is a team assignment" warning banner and render the "unavailable assignment" skeleton.
- Staff views remain unchanged.
- Fix display bug in the `ora_response_unavailable.html` template
- Bump ORA to `2.9.9`

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5280](https://openedx.atlassian.net/browse/EDUCATOR-5280)

## Screenshots
Student who is not on a team for the team-set will see the "unavailable" assignment and a warning banner:

Screen Shot 2020-09-02 at 12 17 43

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Review by jansenk — COMMENTED** [review]
Testing locally but it looks great! I was originally thinking you should add a new "submission" template, but this looks great.

File: openassessment/templates/openassessmentblock/response/oa_response_unavailable.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {% endblock %}
 
 {% block button %}
-    <button class="ui-slidable" aria-expanded="false" disabled>
+    <button class="ui-slidable" aria-expanded="false" disabled></button>
```
[the remark this answers is not in the document]
Comment (nsprenkle) [reply openassessment/templates/openassessmentblock/response/oa_response_unavailable.html:11]: Yes. Both work.

**Review by jansenk — APPROVED** [review]
Looks great. Five stars
